src/lib/create/valid.js

```diff
@@ -5,14 +5,18 @@ import getParsingFlags from '../create/parsing-flags';
 export function isValid(m) {
     if (m._isValid == null) {
         var flags = getParsingFlags(m);
+        var parsedParts = flags.parsedDateParts.some(function (i) {
```

Comment:
Please don't use `some`. Its not supported on IE8 :) You can put it in utils, and use the `Array.prototype` if exists (but don't modify `Array`)